Tests/Tests/STPRedirectContextTest.m

```diff
@@ -297,15 +298,18 @@ - (void)testNativeRedirectSupportingSourceFlow_validNativeURL {
 
     id applicationMock = OCMClassMock([UIApplication class]);
     OCMStub([applicationMock sharedApplication]).andReturn(applicationMock);
+    OCMStub([applicationMock openURL:[OCMArg any]
+                             options:[OCMArg any]
+                   completionHandler:([OCMArg invokeBlockWithArgs:@YES, nil])]);
 
     id mockVC = OCMClassMock([UIViewController class]);
     [context startRedirectFlowFromViewController:mockVC];
 
     OCMReject([sut startSafariViewControllerRedirectFlowFromViewController:[OCMArg any]]);
     OCMReject([sut startSafariAppRedirectFlow]);
-    OCMVerify([applicationMock openURL:[OCMArg any]
-                               options:[OCMArg any]
-                     completionHandler:[OCMArg any]]);
+    OCMVerify([applicationMock openURL:[OCMArg isEqual:sourceURL]
```

Comment:
I think you can just write `sourceURL` instead of `[OCMArg isEqual:sourceURL]` in the verify calls? Not completely sure.

Reply:
Yeah it wasn't clear to me if they were the same. I thought maybe one way did `== sourceURL` and one way did `isEqual:sourceURL`? But I'm not sure.